rero_ils/modules/operation_logs/jsonschemas/operation_logs/operation_log-v0.0.1.json

```diff
@@ -117,6 +117,11 @@
             "ITEM_ON_LOAN",
             "ITEM_RETURNED"
           ]
+        },
+        "library_pid": {
+          "title": "Library PID",
+          "type": "string",
```

Comment:
why not type `keyword`?

Reply:
All other `library_pid` are strings too in the mapping.